packages/core/bootstrap/src/lib/modules/requester.ts

```diff
@@ -48,7 +48,13 @@ export class Requester {
       let response: AxiosResponse<T>
       const url = join(config.baseURL || '', config.url || '')
       try {
+        const startTime = process.hrtime()
+
         response = await axios(config)
+
+        const endTime = process.hrtime(startTime)
+        const milliseconds = Math.round(endTime[0] * 1000 + endTime[1] / 1000000)
```

Comment:
We're doing a network request to an external resource here, so exact ns timings are not going to be very relevant. What's the impact of doing a simple `Date.now()` minus `Date.now()` at the start of the request? Seems simpler to manage, and more readable, right?

Reply:
no particular reason, just thought to be more precise, but i guess `Date.now()` will be more manageable

Reply:
Actually I prefer hrtime, it's meant for these measurements (no idea if it's more performant)

Reply:
Had a closer look in the docs:
> These times are relative to an arbitrary time in the past, and not related to the time of day and therefore not subject to clock drift.

While I do think this one is overkill in terms of complexity, it would also be nice to not be affected by things like clock drift.

Let's keep `process.hrtime`, but let's use `process.hrtime.bigint()`, [as `process.hrtime([time])` is deprecated](https://nodejs.org/docs/latest-v16.x/api/process.html#processhrtimetime).